sdk/attestation/attestation/README.md

```diff
@@ -1,33 +1,261 @@
+a policy management certificate at the time the instance is created. All policy modification operations require that the customer sign
+the policy data with one of the existing policy management certificates. The Policy Management Certificate Management APIs enable
+clients to "roll" the policy management certificates.
+
+### Isolated Mode and AAD Mode
+
+Each Microsoft Azure Attestation service instance operates in either "AAD" mode or "Isolated" mode. When an MAA instance is operating in AAD mode, it means that the customer which created the attestation instance allows Azure Active Directory and Azure Role Based Access control policies to verify access to the attestation instance.  
+
+### *AttestationType*
+
+The Microsoft Azure Attestation service supports attesting different types of evidence depending on the environment.
+Currently, MAA supports the following Trusted Execution environments:
+
+- OpenEnclave - An Intel(tm) Processor running code in an SGX Enclave where the attestation evidence was collected using the OpenEnclave [`oe_get_report`](https://openenclave.io/apidocs/v0.14/enclave_8h_aefcb89c91a9078d595e255bd7901ac71.html#aefcb89c91a9078d595e255bd7901ac71) or [`oe_get_evidence`](https://openenclave.io/apidocs/v0.14/attester_8h_a7d197e42468636e95a6ab97b8e74c451.html#a7d197e42468636e95a6ab97b8e74c451) API.
+- SgxEnclave - An Intel(tm) Processor running code in an SGX Enclave where the attestation evidence was collected using the Intel SGX SDK.
+- Tpm - A Virtualization Based Security environment where the Trusted Platform Module of the processor is used to provide the attestation evidence.
+
+### Runtime Data and Inittime Data
+
+RuntimeData refers to data which is presented to the Intel SGX Quote generation logic or the `oe_get_report`/`oe_get_evidence` APIs. If the caller to the attest API provided a `runtime_data` attribute, The Azure Attestation service will validate that the first 32 bytes of the `report_data` field in the SGX Quote/OE Report/OE Evidence matches the SHA256 hash of the `runtime_data`.
+
+InitTime data refers to data which is used to configure the SGX enclave being attested.
+
+> Note that InitTime data is not supported on Azure [DCsv2-Series](https://docs.microsoft.com/azure/virtual-machines/dcv2-series) virtual machines.
 
-- Microsoft Azure Attestation Enclave Attestation
-- Attestation Policy Management APIs
-- Attestation Policy Management Certificate Management APIs.
+### Additional concepts
 
 ## Examples
 
+- [Create an attestation client instance](#create-client-instance)
+- [Attest an SGX enclave](#attest-sgx-enclave)
+- [Get attestation policy](#get-attestation-policy)
+- [Retrieve token validation certificates](#retrieve-token-certificates)
+- [Create an attestation client instance](#create-client-instance)
+
+### Create client instance
+
+Creates an instance of the Attestation Client at uri `endpoint`.
+
+```ts
+<FILL THIS IN>
```

Comment:
Are these going to be filled in this PR or in upcoming ones?

Reply:
These will be filled in in future pull requests. I don't want to build the snippets until I actually know what the code to implement them is going to look like.

Reply:
Sounds good, perhaps open an issue for it to be part of your release epic so it does not get lost before the release.